### PR TITLE
Changed operator tranlations

### PR DESCRIPTION
**Pull request by map-per**
Changed the translation of the [operator](https://wiki.openstreetmap.org/wiki/DE:Key:operator) key from "Owner" to "Operator". That's clearer and avoids confusion with the [owner](https://wiki.openstreetmap.org/wiki/Key:owner) tag.

### CODE REVIEW

**Comment by biodranik** [comment]
This change may make OM more geeky, and less clear to non-geeks. Most people don't care/never know about the owner of the place, and need to know who operates it. But connecting "operator" with "this is a company/person which runs the place" is difficult, esp. if "owner" is not printed nearby.

**Comment by Misalf-git** [comment]
I don't think that's geeky at all.
It's rather weird (or wrong) to see the operater of ATMs, charging stations or ticket machines displayed as owner.

**Review by vng — APPROVED** [review]
Agree here. The only question, why do we have [operator] and [editor_operator] ?

**Comment by pastk** [comment]
Related to
- https://github.com/organicmaps/organicmaps/pull/8523#issuecomment-2198303514

In many cases `operator` is a very niche information.

File: data/strings/strings.txt [hunk 1]
```diff
@@ -16779,94 +16779,94 @@
...
-    eu = Eragilea: %@
-    fa = مالک: %@
+    et = Operaator: %@
+    eu = Operadorea: %@
+    fa = اپراتور: %@
     fi = Operaattori: %@
-    fr = Opérateur : %@
-    he = בעלים: %@
-    hi = मालिक: %@
+    fr = Opérateur: %@
```
[the remark this answers is not in the document]
Comment (biodranik) [reply data/strings/strings.txt:16844]: Thanks a lot for mentioning it! Fixes are already applied in f40ef7db67

**Pull request merged**